app/src/theme/index.js

```diff
@@ -112,8 +112,8 @@ const theme = {
     border: '1px solid #EEE',
     borderRadius: '8px ',
     boxShadow: '0px 2px 4px rgba(0, 0, 0, 0.05)',
-    paddingHorizontal: '25px',
-    paddingVertical: '25px',
+    paddingHorizontal: '24px',
+    paddingVertical: '20px',
     textColor: '#000',
```

Comment:
@Mi-Lan , if you change here, all paddings of cards will be changed.
Is it ok? @pimato

Reply:
@liaojikunwork @pimato i mean i can overwrite it on component but kinda defeats the purpose of global styles if they are linked here they should be linked everywhere if we want to keep designs consistent...

Reply:
I agree with @Mi-Lan our goal is to get consistent from the design specification to the actual implementation. One issue I see now is that the navigation needs to be adopted according to my designs as part of it is not using the margin-top here:

<img width="486" alt="Bildschirmfoto 2020-11-07 um 10 57 49" src="https://user-images.githubusercontent.com/5337809/98438118-3e05c880-20e8-11eb-8ba9-c46c12bb1508.png">

but I think this should be part of a new issue and PR!